src/games/tic_tac_toe/render/tic_tac_toe.html

```diff
@@ -129,7 +129,7 @@
         this._publicScene = {}
       },
 
-      attached: function() {
+      ready: function() {
```

Comment:
Introduce the usage of ES6 classes for Polymer component definition? Or not? (Polymer2 supports the usage of classes).